### ISSUE TITLE
Fine-tuning on a ImageNet 

### ISSUE THREAD

**Issue by thorpy1992** [BVLC/caffe#5080]
I am trying to fine-tune a model trained on 100000 million ImageNet images.
I changed the final FC layer and renamed it(I have only 2 output classes). I also changed the base lr to 0.0001 and my loss is frozen at 0.69. If I set the base lr at 0.001 the loss is QNAN.
Does anyone knows why?

Thanks in advance!

**Comment by mary-alegro** [comment]
Are you sure your learning parameters aren't frozen?

**Comment by shelhamer** [comment]
From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:

> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
